invenio_app_ils/records/api.py

```diff
@@ -38,11 +41,34 @@
 class IlsRecord(Record):
     """Ils record class."""
 
+    @property
+    def pid(self):
+        """Get the PersistentIdentifier for this record."""
+        return PersistentIdentifier.get(
+            pid_type=self._pid_type,
```

Comment:
Now that I see this, I realize that _pid_type was not a great idea, since we can have multiple pids referencing a record.
Maybe we should have a "main" pid... anyway, nothing to change, just to keep in mind.

Reply:
Something that I would like to see changed related to pid type is renaming all `document_pid`, `keyword_pid`, ... to just `pid` and include `pid_type` in the metadata. This would make it a lot easier to identify the record type when dealing with multiple record types.

Reply:
Good, can we discuss this tomorrow?

Reply:
@equadon why don't you use the schema property to identify the record type? This is what is happening now in videos...I don't remember if there was an important reason not to have a common name...

Reply:
@zzacharo That's what I'm doing now in the UI but it's not as clean in my opinion since the schema is versioned. It would be easier to just match a string to figure out the record type.